build.gradle

```diff
@@ -565,10 +565,15 @@ def jfrWithMem = ["-XX:+UnlockCommercialFeatures", "-XX:+UnlockDiagnosticVMOptio
                   "-XX:StartFlightRecording=delay=2s,duration=60m,name=mem_ex,filename=recording.jfr,settings=profile_heap_exception",
                   "-XX:+FlightRecorder", "-XX:FlightRecorderOptions=disk=true,maxage=10h,dumponexit=true,loglevel=info"]
 
+
```

Comment:
We should document how to use this. Or at least point to the logback documentation

Reply:
Agreed with Justin, otherwise LGTM.